semopenalex-app/data/templates/http%3A%2F%2Fwww.metaphactory.semopenalex.com%2Fapp%2FStart.html

```diff
@@ -445,6 +461,8 @@
         <li><a href="https://www.flaticon.com/free-icons/university" target='_blank'>University icons created by monkik - Flaticon</a></li>
         <li><a href="https://www.flaticon.com/free-icons/location" target='_blank'>Location icons created by Freepik - Flaticon</a></li>
         <li><a href="https://www.flaticon.com/free-icons/lightbulb" target='_blank'>Lightbulb icons created by Freepik - Flaticon</a></li>
+        <li><a href="https://www.flaticon.com/free-icons/stack-of-books" target='_blank'>Books icons created by Freepik - Flaticon</a></li>
```

Comment:
The attribution would need to be fixed, the author is not Freepik in this case, but "Smashicons", see https://www.flaticon.com/free-icon/book-stack_3389081?term=stack+of+books&page=1&position=1&origin=tag&related_id=3389081

Recommended attribution is 

```
<a href="https://www.flaticon.com/free-icons/book" title="book icons">Book icons created by Smashicons - Flaticon</a>
```